_includes/manuals/1.20/1.2_release_notes.md

```diff
@@ -350,5 +352,11 @@ same time. In order to prevent this in future, we decided to add a constraint on
 means that existing duplicates need to be unified manually. Either delete all duplicates from `puppetclasses`
 table or try using [foreman-maintain](https://github.com/theforeman/foreman_maintain) that should fix the issue.
 For more information, see [the redmine issue](http://projects.theforeman.org/issues/24923/)
+* A change introduced to the host fact API in 1.19.0 has led to incompatibilities and has been reverted.
+Any scripts using this API may need to be updated accordingly.
+See ([the redmine issue](https://projects.theforeman.org/issues/25188)) for more details.
+* The ability to define `puppetconfdir`, `puppetvardir` and `puppetssldir` via `settings.yaml` has
```

Comment:
The `ssl_*` settings mentioned can still be set via `settings.yaml` (which the installer does). It's that they previously were derived from `puppetssldir` if unset.

Reply:
updated the text, let me know it it is clearer now. thanks!